services/tone_analyzer/v3-beta.js

```diff
@@ -16,7 +16,7 @@
 
 module.exports = function (RED) {
   var cfenv = require('cfenv');
-
+  
   var services = cfenv.getAppEnv().services,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['services' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2393366918/issues/source?bid=3225115&fileBranchId=3230140#l20)